docs/samples/advanced_config/key_rotator.yaml

```diff
@@ -0,0 +1,42 @@
+database:
+  # Database URL. (required)
+  url: "postgres://postgres:postgres@localhost:5432/postgres"
+
+  # Timeout for new database connections. Defaults to 60 seconds.
+  connection_pool_timeout_secs: 60
+
+  # Maximum number of database connections. Defaults to CPUs * 4.
+  connection_pool_max_size: 8
+
+  # Flag to check if the database schema version is compatible upon startup.
+  # (optional, defaults to true)
+  check_schema_version: true
+
+  # Path to a PEM file with root certificates to trust for TLS database
+  # connections. If present, TLS may be used depending on the "sslmode"
+  # connection string parameter, and the server's support for TLS. If absent,
+  # TLS will never be used. (optional)
+  tls_trust_store_path: /path/to/file.pem
+
+key_rotator:
+  # Rotation policy for global HPKE keys. At least one is required. Each entry
+  # represents a key with a particular ciphersuite.
```

Comment:
I think the second and third sentences here now belong on the `ciphersuite` key below